assets/prototype/domain/eventEditor/containers/queries/useDatetimeIds.js

```diff
@@ -0,0 +1,31 @@
+import * as R from 'ramda';
+import { useApolloClient } from '@apollo/react-hooks';
+import { GET_DATETIMES } from './dates';
+
+const useDatetimeIds = () => {
+	const eventId = R.pathOr(null, ['eeEditorEventData', 'eventId'], window);
+	const client = useApolloClient();
+	let data;
+
+	try {
+		data = client.readQuery({
+			query: GET_DATETIMES,
+			variables: {
+				where: {
+					eventId,
+				},
+			},
+		});
+	} catch (error) {
+		throw new Error(error);
```

Comment:
I don't think throwing an error is enough here. It's possible that there may not be any datetimes in cache, so `readQuery` will throw an error, in that case we should return an empty array.